coverage-tests.js

```diff
@@ -1278,26 +1260,27 @@ test('should send dom and location when check region by selector with custom scr
   env: {browser: 'chrome', args: ['hide-scrollbars']},
   variants: {
     '': {vg: false},
-    'with vg': {vg: true, skipEmit: true}, // TODO grid doesn't return the correct location for this test
+    'with vg': {vg: true,skipEmit: true, skipEmit: true}, // TODO grid doesn't return the correct location for this test
   },
   test({driver, eyes, assert, helpers}) {
     eyes.open({appName: 'Applitools Eyes SDK', viewportSize})
-    driver.executeScript('window.scrollTo(0, 350)') // not really needed
     driver.click('#centered')
-    driver.executeScript('document.documentElement.style.overflow="hidden";') // TODO this is due to differences between JS and non-JS SDK's. Since what's important in this test is not the image itself, nor can this be solved with branch baselines, we hard coded hide scrollbars on the HTML element in order to verify that the DOM and location are correct.
-    driver.executeScript('document.querySelector("#modal-content").setAttribute("data-applitools-target", "true");')
-    eyes.check({region: '#modal-content', isFully: false, scrollRootElement: '#modal1'})
-    const result = eyes.close(false).ref('result')
-    const info = helpers.getTestInfo(result).ref('info')
+    // TODO this is due to differences between JS and non-JS SDK's. Since what's important in this test is not the image itself, nor can this be solved with branch baselines, we hard coded hide scrollbars on the HTML element in order to verify that the DOM and location are correct.
+    // TODO this happens because non-JS emitters are ignore env args
```

Comment:
what do you mean by this?

Reply:
The first `TODO` was left by @itaibh. And in the second one, I explain that the code he adds is not actually needed if non-JS emitters will properly work with `env` configuration. As you can see I specified there `hide-scrollbars` which will start chrome without scrollbars (same as grid does).